server/acl_casbin_policy_developer.csv

```diff
@@ -77,3 +77,5 @@ p, DEVELOPER, /sheet/{id}, GET
 p, DEVELOPER, /sheet/{id}, PATCH
 p, DEVELOPER, /sheet/{id}, PATCH_SELF
 p, DEVELOPER, /sheet/{id}, DELETE_SELF
+p, DEVELOPER, /debug, GET
+p, DEVELOPER, /debug, PATCH
```

Comment:
We may not let developers to change debug setting. Also show debug only only for admins and DBAs.

@tianzhou, wdyt?

Reply:
SGTM